manifests/h/HydrologicEngineeringCenter/HEC-DSSVue/3.2.3/HydrologicEngineeringCenter.HEC-DSSVue.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: HydrologicEngineeringCenter.HEC-DSSVue
+PackageVersion: 3.2.3
+InstallerSwitches:
+  Silent: /s /v/qn
+  SilentWithProgress: /S

```

Silent with progress requires user interaction; Switches should be updated
```suggestion
  SilentWithProgress: /s /v/qb
```